CHANGELOG.md

```diff
@@ -1,10 +1,21 @@
 # Changelog
+
 All notable changes to this project will be documented in this file.
 
 ## [Unreleased]
 
+## [v2.38.0] 2022-09-03
+
```

Comment:
R

Reply:
Thanks for catching it. Autoformatter in Intellij :/